tools/proofers/spellcheck.inc

```diff
@@ -184,57 +150,42 @@ if ($user->profile->i_type == 1) {
     echo "</div>";
     // here ends the div for the WordCheck header
 
-?>
-<table id="tbtext">
-<tr><td id="tdtop">
-<?php
+    echo '<table id="tbtext">';
+
     // print the link to the WordCheck FAQ
+    echo '<tr><td id="tdtop">';
     echo new_window_link(get_faq_url("wordcheck-faq.php"), _("WordCheck FAQ"));
-?>
-</td></tr>
-<tr>
-<td id="tdtext">
-<?php
+    echo '</td></tr>';
+
+    echo '<tr>';
+    echo '<td id="tdtext">';
     $ppage->echo_hidden_fields();
     echo $page_contents;
-?>
-</td>
-</tr>
-<tr>
-    <td id="tdtop">
-        <input
-            type="submit"
-            name="spcorrect"
-            id="spcorrect"
-            value="<?php echo attr_safe(_("Submit Corrections")); ?>"
-            title="<?php echo attr_safe($keep_corrections); ?>"
-        >
-        &nbsp;&nbsp;&nbsp;
-        <input
-            type="submit"
-            name="spexit"
-            value="<?php echo attr_safe(_("Quit WordCheck")); ?>"
-            title="<?php echo attr_safe(_("Abandon corrections and return to proofreading this page")); ?>"
-            onClick="return confirmExit();"
-        >
-        <input
-            type="submit"
-            name="spsaveandnext"
-            id="spsaveandnext"
-            value="<?php echo attr_safe(_("Save as 'Done' & Proofread Next Page")); ?>"
-            title="<?php echo attr_safe(_("Save page as done and proofread the next available page")); ?>"
-<?php
-    if ($is_changed) {
-        echo "            disabled\n";
-    }
-?>
-        >
-    </td>
-</tr>
-</table>
-</form>
-</div>
-<?php
+    echo '</td>';
+    echo '</tr>';
+
+    echo '<tr>';
+    echo '<td id="tdbottom">';
```

Comment:
The ID was changed from `tdtop` to `tdbottom` as `tdtop` already exists above. Found this after running the resulting HTML through the W3C validator to confirm I didn't screw up any of the tags.